lib/rules/indentation/html.test.js

```diff
@@ -679,3 +679,141 @@ testRule({
 		},
 	],
 })
+testRule({
+	plugins,
+	ruleName,
+	config: [
+		2,
+		{
+			baseIndentLevel: 1,
+		},
+	],
+	customSyntax: `postcss-html`,
+	fix: true,
+
+	accept: [
+		{
+			code: stripIndent(`
+				<html>
+				  <head>
+				    <style>
+				      a {
+					display: block;
+				      }
+				    </style>
+				  </head>
+				</html>
+			`),
+		},
+		{
+			code: stripIndent(`
+				<div>
+				  <style>
+				    a {
+				      color: pink;
+				    }
+				  </style>
+				</div>
+			`),
+		},
+	],
+
+	reject: [
+		{
+			code: stripIndent(`
+				<html>
+				  <head>
+				    <style>
+				    a {
+					display: block;
+				      }
+				    </style>
+				  </head>
+				</html>
+			`),
+			fixed: stripIndent(`
+				<html>
+				  <head>
+				    <style>
+				      a {
+					display: block;
```

Comment:
Also, as in the previous comment - we need to improve the implementation so that it takes into account indentation of nested properties, not just the first line